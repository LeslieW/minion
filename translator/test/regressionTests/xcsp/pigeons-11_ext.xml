<?xml version="1.0" encoding="UTF-8"?>

<instance>
<presentation name="?" maxConstraintArity="11" format="XCSP 2.0"/>

<domains nbDomains="1">
<domain name="D0" nbValues="10">0..9</domain>
</domains>

<variables nbVariables="11">
<variable name="V0" domain="D0"/>
<variable name="V1" domain="D0"/>
<variable name="V2" domain="D0"/>
<variable name="V3" domain="D0"/>
<variable name="V4" domain="D0"/>
<variable name="V5" domain="D0"/>
<variable name="V6" domain="D0"/>
<variable name="V7" domain="D0"/>
<variable name="V8" domain="D0"/>
<variable name="V9" domain="D0"/>
<variable name="V10" domain="D0"/>
</variables>

<constraints nbConstraints="1">
<constraint name="C0" arity="11" scope="V0 V1 V2 V3 V4 V5 V6 V7 V8 V9 V10" reference="global:allDifferent"/>
</constraints>
</instance>
