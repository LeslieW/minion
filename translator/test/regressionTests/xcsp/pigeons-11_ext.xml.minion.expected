MINION 3

#This file has been automatically generated by TAILOR v0.2
#bug-reports: andrea@cs.st-and.ac.uk

# amount of common subexpressions used:0
# amount of inferred common subexpressions used:0
# amount of original variables saved (over direct equality):0
# amount of constraints:1
# amount of auxiliary variables: 0

**VARIABLES**
DISCRETE V0 {0..9}
DISCRETE V1 {0..9}
DISCRETE V2 {0..9}
DISCRETE V3 {0..9}
DISCRETE V4 {0..9}
DISCRETE V5 {0..9}
DISCRETE V6 {0..9}
DISCRETE V7 {0..9}
DISCRETE V8 {0..9}
DISCRETE V9 {0..9}
DISCRETE V10 {0..9}

# aliases

**SEARCH**

PRINT [
[V0],[V1],[V2],[V3],[V4],[V5],[V6],[V7],
[V8],[V9],[V10]]

VARORDER [
V0,V1,V2,V3,V4,V5,V6,V7,
V8,V9,V10]


**CONSTRAINTS**

alldiff([ V0, V1, V2, V3, V4, V5, V6, V7, V8, V9, V10])

**EOF**
